gamemodes/terrortown/gamemode/server/sv_networking.lua

```diff
@@ -217,7 +217,7 @@ function SendFullStateUpdate()
 			local rd = v:GetSubRoleData()
 
 			if not roleData.unknownTeam and v:HasTeam(team)
-			or v:GetNWBool("body_found")
+			or v:GetNWLibBool("bodyFound")
 			or team == TEAM_TRAITOR and rd.visibleForTraitors
 			or roleData.networkRoles and table.HasValue(roleData.networkRoles, rd)
```

Comment:
Unrelated to the PR but every occurences of table.HasValue should be replaced by hashtables

Reply:
Thank you, i will give it a look and @Alf21 might too as soon as he got some time